src/App/App.js

```diff
@@ -1,44 +1,57 @@
 import React, { Component } from 'react';
 import './App.scss';
 import Header from '../Header/Header';
-import movieData from '../Data/movieData';
-import fakeMovieDetails from '../Data/fakeMovieDetails';
 import MovieDetails from '../MovieDetails/MovieDetails';
 import tomatillo from '../assets/tomatillo.svg';
 import MovieList from '../MovieList/MovieList';
 import MovieCard from '../MovieCard/MovieCard';
+import { fetchAllMovies, fetchSingleMovie, fetchVideo } from '../Data/apiCalls';
 import logo from '../assets/logo.svg';
-import tomatillo from '../assets/tomatillo.svg';
-
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movieData: movieData.movies,
-      movieDetails: fakeMovieDetails.movie,
-      pageLocation: 0
-      // movieData: [],
-      // movieDetails: [],
-      // filteredMovies: [],
-      // error: ''
+      movieData: [], //all of the movie on home page
+      movieDetails: [], //singe movie
+      filteredMovies: [],
+      pageLocation: 0,
+      // currentMovie: {},
```

Comment:
we may be able to skip this or possibly replace the name of movieDetails to this.

Reply:
That's a great idea!